Astro CLI 1.2 docs (#855)
astronomer/docs: software_versioned_docs/version-0.28/cli-release-notes.md

> If you have any questions or a bug to report, reach out to us via [Astronomer Support](https://support.astronomer.io).

Reviewer: ```suggestion
If you have any questions or a bug to report, contact [Astronomer Support](https://support.astronomer.io).
```

> ## Astro CLI v1.2.0

Reviewer 2: I think we don't need this under 0.28, as 1.2.0 will only be compatible with the 0.29 Software platform version
